docs/release-notes.md

```diff
@@ -1,15 +1,46 @@
 # Lockbox Release Notes
 
+## 0.1.8-alpha
+
+_Date: 2018-03-08_
+
+### What's New
+
+- Added support For Telemetry Scalars so we can easily tell how many items are stored in your Lockbox datastore ([#552](https://github.com/mozilla-lockbox/lockbox-extension/pull/552))
+- Made it so the search box is automatically focused so you can start typing to search your entries immediately ([#573](https://github.com/mozilla-lockbox/lockbox-extension/issues/573))
+- The doorhanger also automatically fills the search with the domain of the current page you're viewing to help you quickly see matching entries ([#616](https://github.com/mozilla-lockbox/lockbox-extension/pull/616))
```

Comment:
\o/

Reply:
@sandysage would you like to acknowledge the "limitations" of the autofill here? or in the known issues section? either way I didn't add anything and leave it to your judgement on what/how we should disclose. 😀

Reply:
@devinreams Yes!

The doorhanger automatically fills the search with the domain, which allows for future improvements to accommodate when there are no results (#515) and parsing the results (#521)...stay tuned.

Reply:
Let's just leave this as is. Everything else checks out and it's not worth the changes here. Besides, my note might just be opening a can of worms unnecessarily.

Reply:
>Let's just leave this as is. E

@sandysage Hah whoops I didn't see this until now... I think we're done here! You can always go back and edit if you'd like. 😁